addons/project/tests/test_personal_stages.py

```diff
@@ -90,6 +90,29 @@ def test_personal_stage_read_group(self):
         self.assertEqual(1, total_stage_0)
         self.assertEqual(1, total_stage_1)
 
+    def test_default_personal_stage(self):
+        user_without_stage = self.env['res.users'].create({
+            'login': 'test_no_stage',
+            'name': "Test User without stage",
+        })
+        user_with_stages = self.env['res.users'].create({
+            'login': 'test_stages',
+            'name': "Test User with stages",
+        })
+        personal_stage = self.env['project.task.type'].create({
+            'name': 'personal stage',
+            'user_id': user_with_stages.id,
+        })
+        # ensure that a user without personal stage is getting the default stages
+        self.task_1.with_user(user_without_stage).action_menu_view_all_task()
+        stages = self.env['project.task.type'].sudo().search([('user_id', '=', user_without_stage.id)])
+        self.assertEqual(len(stages), 7, "The default personal stage were not correctly created for the user")
+        # ensure that the user's personal stages are not changing if the user already had some
+        self.task_1.with_user(user_with_stages).action_menu_view_all_task()
+        stages = self.env['project.task.type'].sudo().search([('user_id', '=', user_with_stages.id)])
+        self.assertEqual(len(stages), 1, "The default personal stages shouldn't be added to the user")
+        self.assertEqual(stages.id, personal_stage.id, "The personal stage of the user should not change")
```

Comment:
```suggestion
        self.assertEqual(stages, personal_stage, "As this user already had a personal stage, none should be added")
```
- One assert is enough.
- I like to give a short explanation.